code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -264,7 +264,7 @@
 //	icon_state = "xburger"												//Refers to an icon in food/food.dmi
 //	New()																//Don't mess with this.
 //		..()															//Same here.
-//		reagents.add_reagent("xenomicrobes", 10)						//This is what is in the food item. you may copy/paste
+//		reagents.add_reagent("nanomachines", 10)						//This is what is in the food item. you may copy/paste
```

Comment:
🤨

Reply:
Well I was going to delete all mentions of it in the code, but things changed, ill fix it